draft-ietf-taps-interface.md

```diff
@@ -1185,30 +1193,41 @@ number of Messages with a single call. This allows an application to provide
 backpressure to the transport stack when it is temporarily not ready to
 receive messages.
 
+Receive takes an optional minIncompleteLength argument. This value indicates
+the smallest partial Message data size that should be delivered. By default, this
+value is infinite, which means that only complete, atomic Messages will be delivered.
+If this value is set to some smaller value, the associated ReceiveHandler will
+be triggered only when at least that many bytes are available, the Message is
+complete, or the system needs to free up memory.
```

Comment:
Since the last case is exceptional, perhaps we should mention that applications must check the received length before use, as it may be less than minIncompleteLength?

Reply:
Added text